client/state/data-layer/wpcom/sites/automated-transfer/status/test/index.js

```diff
@@ -66,7 +71,7 @@ describe( 'receiveStatus', () => {
 	test( 'should request status again if not complete', () => {
 		const dispatch = jest.fn();
 		receiveStatus( { siteId }, IN_PROGRESS_RESPONSE )( dispatch );
-		clock.tick( 4000 );
+		jest.advanceTimersByTime( 4000 );
```

Comment:
I managed to simplify this one by:
- using `jest.useFakeTimers()` only at the beginning of this one specific test
- calling `jest.runAllTimers()` instead of `advanceTimersByTime`. That will trigger the `setTimeout` that was scheduled inside `receiveStatus`, no matter what the timeout period was.

Reply:
I have no preferences and find both to be simple enough, so I've went with your suggestion in 2aa44e262a90befbffc17f57e806accc7c03d812.